index.html

```diff
@@ -67,64 +60,131 @@ <h2>
-				</svg>
+				  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M324.3 80.3c-1.2-.2-2.4-.3-3.6-.3v.4-.3h-.4c-22 0-42.9 25.9-47.8 60.3-5.3 36.6 9.4 69 32.7 72.4 1.3.2 2.7.3 4 .3 22 0 42.9-25.9 47.8-60.3 5.3-36.7-9.4-69.1-32.7-72.5zM320.6 80zM442.3 169.4c-3.5-1.4-7.1-2.1-10.9-2.1h-.4c-19.7.3-41.8 19.2-53.4 47.8-13.9 34.2-7.5 69.2 14.4 78.2 3.5 1.4 7.1 2.1 10.9 2.1 19.8 0 42.2-19 53.9-47.8 13.8-34.2 7.3-69.2-14.5-78.2zM327.6 295.4c-27.8-43.6-39.8-60-71.6-60s-43.9 16.5-71.7 60c-23.8 37.2-71.9 40.3-83.9 71.9-2.4 5.6-3.6 11.7-3.6 18.2 0 25.7 20.8 46.5 46.4 46.5 31.8 0 75.1-24 112.9-24s80.9 24 112.7 24c25.6 0 46.3-20.8 46.3-46.5 0-6.5-1.3-12.6-3.7-18.2-12-31.7-60-34.7-83.8-71.9zM202.8 213c1.3 0 2.7-.1 4-.3 23.4-3.4 38-35.8 32.7-72.4-5-34.5-25.9-60.3-47.8-60.3-1.3 0-2.7.1-4 .3-23.4 3.4-38 35.8-32.7 72.4 5 34.4 25.9 60.3 47.8 60.3zM120.1 293.3c21.9-9 28.3-44 14.4-78.2-11.7-28.8-34.1-47.8-53.9-47.8-3.8 0-7.4.7-10.9 2.1-21.9 9-28.3 44-14.4 78.2 11.7 28.8 34.1 47.8 53.9 47.8 3.8 0 7.4-.7 10.9-2.1z"/>
+				  </svg>
           	<p>
           		Lorem ipsum dolor sit amet,
           	</p>
           </div>
-
         </section>
 
         <!-- Second Section -->
-        <section></section>
+        <section>
+          <div>
+            <img src="https://via.placeholder.com/1200x450" alt="placeholder">
+          </div>
+        </section>
 
         <!-- Third Section -->
-        <section></section>
+        <section>
+          <article>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit cursus risus at ultrices mi. Volutpat odio facilisis mauris sit amet massa vitae tortor condimentum. Auctor neque vitae tempus quam. 
+            </p>
+          </article>
+          <article>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit cursus risus at ultrices mi. Volutpat odio facilisis mauris sit amet massa vitae tortor condimentum. Auctor neque vitae tempus quam. 
+            </p>
+          </article>
+
+          <!-- button -->
+          <div> 
+            <a href="readmore.html">Read More</a>
+          </div>
+        </section>
+
+      <!-- Fourth Section -->
+      <section>
+        <figure>
+          <a href="pageA.html" title="go to pageA">
+           <img src="https://via.placeholder.com/200x200" alt= "placeholder"/>
+          </a>
+          <figcaption>Lorem ipsum</figcaption>
+        </figure>
+        <figure>
+          <a href="pageB.html" title="go to pageB">
+           <img src="https://via.placeholder.com/200x200" alt= "placeholder"/>
+          </a>
+          <figcaption>Lorem ipsum</figcaption>
+        </figure>
+        <figure>
+          <a href="pageC.html" title="go to pageC">
```

Comment:
the other pages names should be more semantic. e.g about.html, contact.html, prices.htlm